start_end-to-end-test.sh

```diff
@@ -39,7 +39,7 @@ end_to_end_test()
 
 }
 
-end_to_end_test "$@"
+end_to_end_test "$@" && \
 echo "\n\n           End-to-end tests ended successfully! :-)\n\n"
```

Comment:
this is not needed because the script has the `errexit` (-e) flag set

Reply:
Oh yes, that's redundant then.